apps/i18n/dance/en_us.json

```diff
@@ -27,6 +18,15 @@
   "danceFeedbackUseSetSize": "Use the `set backup_dancer2 size` block to make that dancer smaller.",
   "danceFeedbackUseSetTint": "Use the `set tint` block to change a dancer's tint.",
   "danceFeedbackUseStartMapping": "Try adding the <xml><block type=\"Dancelab_startMapping\"><title name=\"SPRITE\">right_unicorn</title><title name=\"PROPERTY\">\"scale\"</title><title name=\"RANGE\">\"bass\"</title></block></xml> block to your program.",
-  "danceFeedbackKeepCoding": "Keep coding! Something's not quite right yet.",
-  "measure": "Measure:"
+  "measure": "Measure:",
+  "DanceAiNewMove": "⚠️ You need to add a block to make the dancer do a new move.",
+  "danceAIFeedbackNeedNewDancer": "⚠️ You need to make a dancer.",
+  "DanceAiFeedbackAiBlockNoParameters": "⚠️ Choose 3 emojis for AI to generate effects.",
+  "DanceAiFeedbackAiBlockNotConnectedSetup": "⚠️ Connect the `AI generate effect` block to the `setup` block.",
+  "DanceAIFeedbackNoAiBlock": "⚠️ Add the `AI generate effect` block to setup.",
+  "DanceAICustomize": "⚠️ You need to change the effects. You can either regenerate the effects, start over with new emojis, or customize the code A.I. created.",
+  "DanceAIFeedbackConnectPressUpArrowEvent": "⚠️ Press up arrow ▲ button after pressing *\"Run\"*.",
+  "DanceAIFeedbackKeyPress": "⚠️ You pressed a key, but your dancer didn't do a new move.",
+  "DanceAiFeedbackAiBlockNoParametersOnEvent": "⚠️ The effects didn't change after 2 measures. Choose 3 emojis that match the mood of the song for AI to generate your new stage.",
+  "DanceAIStageChangeAfterMeasures": "⚠️ Connect the `AI generate effect` block to the `After 2 Measures` event."
```

Comment:
`After` should be `after` if we want to match the block text.

Reply:
Already fixed!